internal/cluster/validations/validations.go

```diff
@@ -348,7 +349,7 @@ func ValidateNoProxyFormat(noProxy string) error {
 		}
 		return errors.Errorf("NO Proxy format is not valid: '%s'. "+
 			"NO Proxy is a comma-separated list of destination domain names, domains, IP addresses or other network CIDRs. "+
-			"A domain can be prefaced with '.' to include all subdomains of that domain.", noProxy)
+			"A domain can be prefaced with '.' to include all subdomains of that domain. Use '*' to bypass proxy for all destinations.", noProxy)
```

Comment:
Shouldn't the addition `Use '*' to bypass proxy for all destinations."` be removed since it cannot be used once this PR is merged ?

Reply:
I though about it, but am afraid we'll forget to bring it back :( Right now it's not ideal, but at least the error message when blocking `*` makes it clear that the value is supposed to be valid, just not supported in current release. The doc in the UI will change though.